### PR TITLE
Builder API improvement (#921)

### PR DESCRIPTION
**Pull request by dinhlongviolin1**
Solve Issue #921

### CODE REVIEW

File: src/taipy/gui/_renderers/_class_api/element_api_generator.py [hunk 1]
```diff
@@ -65,30 +76,44 @@ def add_library(self, library: "ElementLibrary"):
...
-                _ElementApiGenerator().createControlElement(element_name, f"{library_name}.{element_name}"),
+                _ElementApiGenerator().createControlElement(
+                    element_name, f"{library_name}.{element_name}", element.default_attribute
+                ),
             )
 
     @staticmethod
-    def createBlockElement(classname: str, element_name: str):
-        return _ElementApiGenerator.createElementApi(classname, element_name, BlockElementApi)
+    def createBlockElement(
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/_renderers/_class_api/element_api_generator.py]: camel case in python ?

**Review by FredLL-Avaiga — APPROVED** [review]
BIG changes

File: src/taipy/gui/_renderers/_class_api/element_api_generator.py (hunk 1 above)
Comment (dinhlongviolin1) [reply src/taipy/gui/_renderers/_class_api/element_api_generator.py]: lol you are right :D

**Review by FabienLelaquais — APPROVED** [review]
Looking good!

**Pull request merged**